### ISSUE TITLE
ms-swift训练Qwen3-VL时不能使用文本-图文混合语料吗？

### ISSUE THREAD

**Issue by alanayu** [modelscope/ms-swift#6795]
我使用文本-图文混合语料(比例1：1）训练Qwen3-VL-32B-Instruct模型，发现训练在第一步就会卡住。
但是，分别使用文本语料或图文语料训练Qwen3-VL-32B-Instruct，训练可以正常进行。

训练GPU是  8XH200
训练脚本是

export CUDA_VISIBLE_DEVICES=0,1,2,3,4,5,6,7 
export NPROC_PER_NODE=8 
export PYTORCH_CUDA_ALLOC_CONF='expandable_segments:True'

export ROOT_IMAGE_DIR=/mnt/nvme1/liufei/new_VLM/vlm_dataset/InternVL1.5_dataset/playround/data/
export IMAGE_MAX_TOKEN_NUM=1024 
export VIDEO_MAX_TOKEN_NUM=128 
export FPS_MAX_FRAMES=16 

MODEL_PATH=Qwen/Qwen3-VL-32B-Instruct
DATA_PATH=/the/path/of/data.jsonl

swift sft \
    --model ${MODEL_PATH} \
    --dataset ${DATA_PATH} \
    --load_from_cache_file true \
    --train_type full \
    --split_dataset_ratio 0 \
    --torch_dtype bfloat16 \
    --num_train_epochs 1 \
    --per_device_train_batch_size 4 \
    --per_device_eval_batch_size 1 \
    --gradient_accumulation_steps 8 \
    --attn_impl flash_attn \
    --padding_free true \
    --learning_rate 1e-6 \
    --freeze_vit true \
    --freeze_aligner true \
    --packing false \
    --lazy_tokenize true \
    --gradient_checkpointing true \
    --vit_gradient_checkpointing false \
    --eval_strategy no \
    --save_steps 200 \
    --save_total_limit 1 \
    --logging_steps 1 \
    --max_length 20480 \
    --output_dir outputs/Qwen3-VL-32B-Instruct-pretrain-$(date '+%Y%m%d') \
    --warmup_ratio 0.05 \
    --deepspeed zero3 \
    --dataset_num_proc 4 \
    --dataloader_num_workers 4 \
    --save_only_model true \
    --seed 666666

麻烦路过的大佬好心人帮忙看看

**Comment by Jintao-Huang** [comment]
这个可以吗：https://github.com/modelscope/ms-swift/blob/main/examples/models/qwen3_vl/mixed.sh

**Comment by alanayu** [comment]
> 这个可以吗：https://github.com/modelscope/ms-swift/blob/main/examples/models/qwen3_vl/mixed.sh

这个lora训练的脚本试过了，不可以

试验的swift版本是3.9.0

**Comment by alanayu** [comment]
用了swift 3.10.2可以了
